src/flex/agentApp/src/WatsonAssistantAgentAppPlugin.tsx

```diff
@@ -0,0 +1,57 @@
+/**
```

Comment:
And there are lint errors in here too.

Reply:
Ended up disabling eslint for these specific Twilio module lines, there's a lot of work the Twilio plugin CLI does that is not clear to me